tests/infer/test_enum.py

```diff
@@ -185,31 +182,27 @@ def test_gmm_batch_iter_discrete_traces(model, data_size, graph_type):
     assert len(traces) == 2
 
 
-@pytest.mark.parametrize("trace_graph", [False, True], ids=["dense", "flat"])
 @pytest.mark.parametrize("model,guide", [
     (gmm_model, gmm_guide),
     (gmm_batch_model, gmm_batch_guide),
 ], ids=["single", "batch"])
-@pytest.mark.parametrize("enum_discrete", [None, "sequential", "parallel"])
-def test_svi_step_smoke(model, guide, enum_discrete, trace_graph):
+@pytest.mark.parametrize("enumerate_default", [None, "sequential", "parallel"])
+def test_svi_step_smoke(model, guide, enumerate_default):
     pyro.clear_param_store()
     data = Variable(torch.Tensor([0, 1, 9]))
 
-    guide = config_enumerate(guide, default=enum_discrete)
+    guide = config_enumerate(guide, default=enumerate_default)
     optimizer = pyro.optim.Adam({"lr": .001})
     inference = SVI(model, guide, optimizer, loss="ELBO",
-                    trace_graph=trace_graph, max_iarange_nesting=1)
+                    enum_discrete=True, max_iarange_nesting=1)
```

Comment:
do you really want ```enum_discrete=True``` to be fixed?

Reply:
Yes, I'm only testing `TraceEnum_ELBO` in this file. I think we should more tests in `tests/infer/test_gradient.py` to test agreement among all ELBO implementations. That will be easier to do once we support `iarange` again in `Trace_ELBO`, since then we can hand-parallelize over number of particles and write lots of cheap tests.